src/application/wizard/service.ts

```diff
@@ -153,7 +156,7 @@ export class WizardService {
         });
```

Comment:
Upload progress should include this buffer stage too. Not sure what the ratio of buffer to s3 upload should be but maybe 50/50?

Reply:
This is being read from a os.tmpDir. Not sure what value this would add.

Reply:
We're awiting this so the client will sit at 0% progress until the file has fully streamed to the server

Reply:
This will have ux implications for low bandwidth users

Reply:
Isn't this just broken down into stages:
1. Client to server, you can track chunks sent (which gives you a percentage process)
2. Server to S3, where you can get updates via the socket

Reply:
Yes, ideally the user will get some feedback about each of these steps as a % of the whole process though. It's something we did with xpub just for buffering prior to he predictor algorithm

Reply:
I'll validate this to be sure.

Reply:
This is what xPub does - https://github.com/elifesciences/elife-xpub/blob/b613112736a3ef8abc60487f699373f17b11a833/packages/component-submission/server/use-cases/helpers/manuscript.js#L44

Reply:
Reviewed the source code of https://www.npmjs.com/package/fs-capacitor. In theory, you can access a readstream to the file in os.tmpdir as it's streaming the file there. Which is actually what the current master branch of graphql-upload does. So it could give more meaningful feedback than the current implementation

Reply:
I suggest doing this as another ticket for two reasons: 
1. The client subscription shouldn't change. 
2. This is closer to xpub besides the predicator being missing - https://github.com/elifesciences/elife-xpub/blob/b613112736a3ef8abc60487f699373f17b11a833/packages/component-submission/server/use-cases/helpers/manuscript.js#L44

Reply:
The change to do this shouldn't be too hard, but it will likely require handing the multipart manually.

Reply:
I agree with @norisalsaadie to do this as part of another ticket.

Another idea would be to send back to the client percentage of upload to s3 and whether the sciencebeam is done or not (since we don't hvae progress for sciencebeam).